Project-iOS/XLProjectName/XLProjectName/Helpers/Extensions/UIRefreshControl+Rx.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  UIRefreshControl+Rx.swift
+//  XLProjectName
+//
+//  Created by Diego Medina on 3/29/16.
+//  Copyright © 2016 XLOrganizationName. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import RxCocoa
+import UIKit
+
+extension UIRefreshControl {
```

Comment:
I thought of that, but for UIControl you should have something more generic (i.e. for any event). But, for the UIRefreshControl, i think observing the ValueChanged Event is the most common thing to do.